backend/src/middlewares/validation/validation.middleware.ts

```diff
@@ -0,0 +1,23 @@
+import { RequestHandler } from 'express';
+import { Schema } from '~/common/types/types';
+import { HttpError } from '~/exceptions/exceptions';
+import { HttpCode } from '~/common/enums/enums';
+
+export const validate = (schema: Schema): RequestHandler => {
```

Comment:
```suggestion
const validateSchema = (schema: Schema): RequestHandler => {
```
Exports only at the bottom of file